wrapanapi/google.py

```diff
@@ -226,15 +226,25 @@ def create_image(self, image_name, bucket_url, timeout=360):
         }
         operation = images.insert(project=self._project, body=data).execute()
         wait_for(lambda: self._nested_operation_wait(operation['name'], zone=False), delay=0.5,
-            num_sec=timeout, message=" Creating image {}".format(image_name))
+                 num_sec=timeout, message=" Creating image {}".format(image_name))
+
+    def delete_image(self, image_name, timeout=360):
+        """ Delete image
+        Args:
+            image_name: Unique name of image
+            timeout: time to wait for operation
+        """
+        operation = self._compute.images().delete(project=self._project, image=image_name).execute()
+        wait_for(lambda: self._nested_operation_wait(operation['name'], zone=False), delay=0.5,
```

Comment:
**REQUIRED:** I'm pretty sure you don't need the **lambda:**.

Reply:
Avoid using lambda.   All it needs is a callable there.   The lambda is unnecessary.   I checked with @psav earlier to make sure.   It's a pretty common comment I've seen other reviewers give, but I needed to get my head around why, which is why I posed the comment so uncertainly.   Pretty certain now.